packages/checkbox/src/css/index.js

```diff
@@ -80,37 +80,37 @@ export default {
   '.psds-checkbox__square--error:after': {
     background: core.colors.red
   },
-  '.psds-checkbox__square--checked': {
+  '.psds-checkbox__square--active': {
```

Comment:
As I've looked at this, first I wanted to keep this as `checked`.  I see that it backs checked and indeterminate states now.  `active` is a somewhat odd name for this, but I don't have a better name.  🤷‍♂  Dunno if you had the same struggle.  "Active" really probably is the generic best.  It matches the lots of other DS components too.